packages/sync-actions/src/product-actions.js

```diff
@@ -204,15 +210,21 @@ export function actionsMapPrices (diff, oldObj, newObj) {
 
   if (variants)
     forEach(variants, (variant, key) => {
-      const [ a, c, r ] = _buildVariantPricesAction(
-        variant.prices,
-        oldObj.variants[key],
-        newObj.variants[key],
-      )
-
-      addPriceActions = addPriceActions.concat(a)
-      changePriceActions = changePriceActions.concat(c)
-      removePriceActions = removePriceActions.concat(r)
+      const {
+        oldObj: oldVariant,
+        newObj: newVariant,
+      } = _extractHashMap(variantHashMap, key, oldObj.variants, newObj.variants)
+      if (REGEX_UNDERSCORE_NUMBER.test(key) || REGEX_NUMBER.test(key)) {
+        const [ a, c, r ] = _buildVariantPricesAction(
```

Comment:
...why don't we call them `additionalAddedPriceActions` etc?

Reply:
that's a good questions, met it like that, but it can be improved